### DISCUSSION TITLE
draggedFiles prop on the returned hook (useDropzone) state has been removed - Any alternatives?

### DISCUSSION THREAD

**Discussion by naemtl** [react-dropzone/react-dropzone#1308] [Q&A]
So, as of v14.0.0, `the {draggedFiles} prop on the returned hook state has been removed and the accept/reject state is only computed on drag enter and drop.`

In my project, I made use of the `draggedFiles` prop and would like to know if there's another prop or value that I can use that would offer me the same information. As I understand it, `acceptedFiles` and `fileRejections` props are populated AFTER drag operations and the `draggedFiles` prop was populated DURING drag operations. Please correct me if I'm wrong.

**Comment by rolandjitsu** [comment]
@naemtl you could use the `onDragEnter`/`onDragLeave`/`onDragOver` callbacks. I cannot recall the reasoning behind removing that property, but I think it must have been:
1. It triggered too many re-renders
2. During drag, there's not a lot of useful information (I think file name is the only data available as far as I recall)
